src/dashboard/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use the official Node.js image as the base image
+FROM node:18-alpine
```

Comment:
I believe there were some steps that needed to be documented

Reply:
Do you mean on how to configure the settings? I added them in the PR description. I'll add them to the readme.